designsafe/apps/data/views/base.py

```diff
@@ -199,12 +199,27 @@ def get_context_data(self, **kwargs):
         elif pub.project.to_dict().get('doi') != None: #This is for older publications
             context['doi'] = pub.project.doi
         context['keywords'] = pub.project.value.keywords.split(',')
-        context['authors'] = [{
-            'full_name': '{last_name}, {first_name}'.format(
-                last_name=user['last_name'], first_name=user['first_name']
-            ),
-            'institution': getattr(getattr(user, 'profile'), 'institution', '')
-        } for user in getattr(pub, 'users', [])]
+        if pub.__contains__('users'):
```

Comment:
Do you mind writing these `__contains__` statements as `if users in pub`?  It's a little more pythonic and readable.

Reply:
Also on line 209.

Reply:
I used __contains__ because when I try the `in` operator, I get an error from Django saying that the BaseESPublication model is not iterable. Would using the `to_dict()` method be okay?